packages/react-kit/src/components/form/Field.styles.ts

```diff
@@ -188,13 +189,28 @@ export const FileUploadWrapper = styled.div<{
     cursor: not-allowed;
     opacity: 0.5;
   }
+  ${({ $v2 }) =>
+    $v2 &&
+    css`
+      aspect-ratio: 1/1;
+      padding-inline: 1.5rem;
+      background: ${colors.white};
+      min-width: 9.375rem;
+      width: auto;
+      height: auto;
+      border-radius: 12px;
+      border: 2px solid ${theme.colors.light.border};
+      color: ${theme.colors.light.greyText};
+    `}
```

Comment:
```suggestion
  ${({ theme : {overrides}) => overrides}
```

Reply:
theme.overrides should be what css`` returns which is`RuleSet<object>`

and these styles should be passed in the Upload component in fermion react-kit